src/index.js

```diff
@@ -94,6 +100,7 @@ const AppContainer = () => {
                 localesManager={localesManager}
                 messages={messages_ref}
                 logo={logo}
+                {...(whiteLogo !== defaultLogo && { whiteLogo })}
```

Comment:
@sniedzielski I saw https://github.com/openimis/openimis-fe-core_js/pull/271/files has handling of switching between logo and whiteLogo – wouldn't it be cleaner to bring that logic in here to keep all the logo switching/determination logic in one place and only pass `logo={logo}` to App? essentially just replace the logo & whiteLogo definition lines with below and get rid of `whiteLogo`

```
const logo = config?.["fe-core"]?.logo?.white ? getConfiguredLogo(appState.config, "white") : getConfiguredLogo(appState.config);
```